app/lib/frontend/templates/misc.dart

```diff
@@ -107,39 +112,77 @@ String renderMiniList(List<PackageView> packages) {
 /// Renders the tags using the pkg/tags template.
 String renderTags(
   List<String> platforms, {
+  @required SearchQuery searchQuery,
+  @required List<String> tags,
   @required bool isAwaiting,
   @required bool isDiscontinued,
   @required bool isLegacy,
   @required bool isObsolete,
   String packageName,
 }) {
-  final List<Map> tags = <Map>[];
+  final List<Map> tabValues = <Map>[];
   if (isAwaiting) {
-    tags.add({
+    tabValues.add({
       'status': 'missing',
       'text': '[awaiting]',
       'title': 'Analysis should be ready soon.',
     });
   } else if (isDiscontinued) {
-    tags.add({
+    tabValues.add({
       'status': 'discontinued',
       'text': '[discontinued]',
       'title': 'Package was discontinued.',
     });
   } else if (isObsolete) {
-    tags.add({
+    tabValues.add({
       'status': 'missing',
       'text': '[outdated]',
       'title': 'Package version too old, check latest stable.',
     });
   } else if (isLegacy) {
-    tags.add({
+    tabValues.add({
       'status': 'legacy',
       'text': 'Dart 2 incompatible',
       'title': 'Package does not support Dart 2.',
     });
+  } else if (requestContext.isExperimental) {
+    if (searchQuery?.sdk == null) {
+      tabValues.addAll(
+        tags.where((s) => s.startsWith('sdk:')).map(
+          (tag) {
+            final value = tag.split(':').last;
+            return {
+              'text': value,
+              'href': urls.searchUrl(sdk: value),
+              'title': tag,
+            };
+          },
+        ),
+      );
+    } else {
+      String prefix;
+      if (searchQuery.sdk == SdkTagValue.dart) {
+        prefix = 'runtime:';
+      } else if (searchQuery.sdk == SdkTagValue.flutter) {
+        prefix = 'platform:';
+      }
+      if (prefix != null) {
+        tabValues.addAll(
+          tags.where((s) => s.startsWith(prefix)).map(
```

Comment:
Some helper function like:
```dart
List<String> tagSuffixes(List<String> tags, String prefix) {
  ... // also sort tags (maybe)
} 
```

might be nice...

Reply:
I'll add a TODO for sorting, right now it will use the pana-provided order. I'd defer the method for a later point, I'd want to refactor this a bit more...